### PR TITLE
[WIP] Initial rails app, setting up tests and CI

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: nested, compact
+Style/ClassAndModuleChildren:
+  Exclude:
+    - 'test/test_helper.rb'
+
+# Offense count: 4
+Style/Documentation:
+  Exclude:
+    - 'spec/**/*'
```
[the remark this answers is not in the document]
Comment (adamruzicka) [reply .rubocop_todo.yml]: How about we drop the `Style/Documentation` cop completely? @ShimShtein @bastilian opinions?

[the remark this answers is not in the document]
Comment (ShimShtein) [reply .rubocop_todo.yml]: I'm OK with dropping it. I don't feel it's useful anyway.

**Comment by adamruzicka** [comment]
Updated

**Comment by ShimShtein** [comment]
Thanks @adamruzicka! Now we have a good place to start. We can tweak the behavior as we roll.

**Pull request merged**